clusterloader2/testing/load/config.yaml

```diff
@@ -17,20 +17,37 @@
 {{$smallRCsPerNamespace := DivideInt $smallRCs $namespaces}}
 {{$totalPodsRounded := AddInt (MultiplyInt $bigRCs $BIG_GROUP_SIZE) (MultiplyInt $mediumRCs $MEDIUM_GROUP_SIZE) (MultiplyInt $smallRCs $SMALL_GROUP_SIZE)}}
 # saturationQPS - qps for RC operations should be an equivalent to 10 [pods/s].
-{{$saturationQPS := DivideFloat (AddInt $bigRCs $mediumRCs $smallRCs) (DivideFloat $totalPodsRounded 10)}}
+{{$bigSaturationQPS := DivideFloat $bigRCs (DivideFloat $totalPodsRounded 10)}}
+{{$mediumSaturationQPS := DivideFloat $mediumRCs (DivideFloat $totalPodsRounded 10)}}
```

Comment:
TBH, I don't think it's exaclty what we want.
I think we should go a different direction to make the test much simpler.

What we know is that we would like to generate average 10pods/s throughput.
So I would suggest introducing a different TuningSet type like:
RandomizedLoad
but where you pass total time over which you spread operations, not the QPS.

That would make it waaaay simpler, because all you will need is one tuningset, not one tunningset per type.

Reply:
> That would make it waaaay simpler, because all you will need is one tuningset, not one tunningset per type.

I guess I have to disagree. If you want to spreed something randomly, you also have to know the number of action to be spread.
Here I see three options:
 - Tuning set takes time and number of operations as a parameters. Still separate tuning sets will be required.
- Add options to have tuning set on step level. It propagates to phases. This one tuning set will be shared. (Still requires previous point)
- Reimplement tuning sets so it takes list of operation to run. (I'm not a fan of this idea though.)

Reply:
> Reimplement tuning sets so it takes list of operation to run. (I'm not a fan of this idea though.)

This is what I had on my mind.
Why do you say it's a bad idea?

Reply:
> Why do you say it's a bad idea?

It's not a bad idea. It will just require reimplementing all tuning sets and how the phases are handled and this will take time.